semgrep/setup.py

```diff
@@ -86,13 +86,11 @@ def run(self):
             os.putenv("SKIP_NUITKA", "TRUE")
             if "osx" in distutils.util.get_platform():
                 with chdir(repo_root):
-                    os.system(  # nosem: contrib.dlint.dlint-equivalent.insecure-os-exec-use
-                        os.path.join(repo_root, "release-scripts/osx-release.sh")
-                    )
+                    os.system(os.path.join(repo_root, "release-scripts/osx-release.sh"))
```

Comment:
I added an ignore for `setup.py` in the linting configuration instead of inline ignores, it seems like more trouble than it's worth at this point.